docs/resources/dms_rocketmq_topic.md

```diff
@@ -54,19 +50,22 @@ The following arguments are supported:
 
 * `total_write_queue_num` - (Optional, Int) Specifies the total number of write queues.
 
-<a name="DmsRocketMQTopic_BrokerRef"></a>
-The `BrokerRef` block supports:
+<a name="dms_arg_brokers"></a>
+The `brokers` block supports:
 
 * `name` - (Optional, String) Indicates the name of the broker.
 
-## Attributes Reference
+## Attribute Reference
 
 In addition to all arguments above, the following attributes are exported:
 
 * `id` - The resource ID.
-  
-<a name="DmsRocketMQTopic_BrokerRef"></a>
-  The `BrokerRef` block supports:
+
+* `brokers` - Specifies the list of associated brokers of the topic.
+  The [brokers](#dms_attr_brokers) structure is documented below.
+
+<a name="dms_attr_brokers"></a>
+  The `brokers` block supports:

```

Comment:
No need to describe here.

Reply:
For the convenience of subsequent parsing, so modifications are made uniformly

Reply:
ok, i got it